.github/workflows/ci.yml

```
@@ -30,12 +34,175 @@ jobs:
+
+      - name: Build the tools (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          # Create obj directory to match expected artifact path
+          New-Item -ItemType Directory -Force -Path "obj"
+
+          # Setup Visual Studio environment and call vcvars to set paths
+          $vsPath = "${env:ProgramFiles}\Microsoft Visual Studio\2022\Enterprise"
+          $devCmd = "$vsPath\Common7\Tools\VsDevCmd.bat"
+          $msbuild = "$vsPath\MSBuild\Current\Bin\MSBuild.exe"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)